### PR TITLE
docs: s3 docs

### PR DESCRIPTION
**Pull request by rephus**
Closes: #4694
Closes: #4693

### Description:

https://deploy-preview-4841--peaceful-bassi-cbf284.netlify.app/guides/enable-cloud-storage

### CODE REVIEW

**Comment by TuringLovesDeathMetal** [comment]
@rephus in my latest PR, I changed the file to `roles.mdx`!

So you might need to rebase?

File: docs/docs/guides/enable-cloud-storage.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Enable cloud storage on Lightdash
```
Comment (TuringLovesDeathMetal) [inline docs/docs/guides/enable-cloud-storage.mdx:1]: @rephus is this just for self-hosted? if yes, can we update the title to "Enable.... (Self-Hosted)"

**Review by TuringLovesDeathMetal — COMMENTED** [review]
one tiny question!

File: docs/docs/guides/enable-cloud-storage.mdx (hunk 1 above)
Comment (rephus) [reply docs/docs/guides/enable-cloud-storage.mdx:1]: it is mainly for self-hosted, because we have s3 enabled on our cloud

Comment (TuringLovesDeathMetal) [reply docs/docs/guides/enable-cloud-storage.mdx]: cool, should we add `Self-Hosted` then in brackets? 

Or add a short description of how this is mostly for self-hosted (and in which cases you should do this if you're on cloud)?

Just so people know whether or not the doc applies to them :)

**Review by TuringLovesDeathMetal — APPROVED** [review]
Looks great! Nice work :)

**Pull request merged**